### PR TITLE
ado-6068 - adding new images and adjusting padding on white buttons

### PR DESCRIPTION
**Pull request by sineade**
https://dev.azure.com/AMA-Ent/AMA-Ent/_workitems/edit/6068

A couple small styling changes for the above card. I needed to add charcoal text as an option for our white button groups, and I also removed some padding, which I tested on our other white buttons and it didn't break them, but did make the buttons I made look better. 

Changes can be seen on staging: https://stage-auth.ama.ab.ca/members/credit_cards

### CODE REVIEW

File: app/assets/stylesheets/layout_components/buttons/button-grouping.scss [hunk 1]
```diff
@@ -153,6 +152,10 @@
     &--blue {
       color: $brand-blue-light;
     }
+
+    &--charcoal {
+      color: $charcoal;
+    }
```
Comment (DaveVanVliet) [inline app/assets/stylesheets/layout_components/buttons/button-grouping.scss]: Hey Sinead, what's the reason for using `charcoal` as the text colour in these buttons, and not `slate` like we use for the majority of our text?

File: app/assets/stylesheets/layout_components/buttons/buttons.scss [hunk 2]
```diff
@@ -68,6 +68,20 @@
     }
   }
 
+  &--white {
```
Comment (DaveVanVliet) [inline app/assets/stylesheets/layout_components/buttons/buttons.scss]: Do you have an example of where this is used @sineade?

File: app/assets/stylesheets/layout_components/buttons/button-grouping.scss (hunk 1 above)
Comment (sineade) [reply app/assets/stylesheets/layout_components/buttons/button-grouping.scss]: lol cause I screwed up

File: app/assets/stylesheets/layout_components/buttons/buttons.scss (hunk 2 above)
Comment (sineade) [reply app/assets/stylesheets/layout_components/buttons/buttons.scss]: * spoke to dave about this one in person

File: app/assets/stylesheets/layout_components/buttons/button-grouping.scss (hunk 1 above)
Comment (DaveVanVliet) [reply app/assets/stylesheets/layout_components/buttons/button-grouping.scss]: looks good!

**Pull request merged**